tests/baselines/reference/duplicatePackage_relativeImportWithinPackage.trace.json

```diff
@@ -0,0 +1,45 @@
+[
+    "======== Resolving module 'foo/use' from '/index.ts'. ========",
+    "Module resolution kind is not specified, using 'NodeJs'.",
+    "Loading module 'foo/use' from 'node_modules' folder, target file type 'TypeScript'.",
+    "Found 'package.json' at '/node_modules/foo/package.json'.",
+    "File '/node_modules/foo/use.ts' does not exist.",
+    "File '/node_modules/foo/use.tsx' does not exist.",
+    "File '/node_modules/foo/use.d.ts' exist - use it as a name resolution result.",
+    "Resolving real path for '/node_modules/foo/use.d.ts', result '/node_modules/foo/use.d.ts'.",
+    "======== Module name 'foo/use' was successfully resolved to '/node_modules/foo/use.d.ts'. ========",
```

Comment:
this is not related to this PR, but just a thought that came to me, we are not logging the packageid anywhere. we should probably to make these tests more statically verifiable

Reply:
#21232